nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -1,14 +1,45 @@
 const express = require("express");
 const router = express.Router();
 
-const meals = require("./../data/meals.json");
+const meals = require("../data/meals.json");
 
 router.get("/", async (request, response) => {
-  try {
-    console.log(meals);
-    console.log("in /api/meals");
-  } catch (error) {
-    throw error;
+  console.log(meals);
+  console.log("in /api/meals");
+  const query = request.query
+  console.log(request.query)
+  if ((req.query.maxPrice && isNaN(parseInt(req.query.maxPrice))) ||
+    (req.query.createdAfter && isNaN(Date.parse(req.query.createdAfter))) ||
+    (req.query.limit && isNaN(parseInt(req.query.limit)))
+  ) {
+    return res.status(400).send("Given parameter is not supported")
```

Comment:
'Res' is not defined. Should be changed to 'response', since this is what you wrote in line 6.

Reply:
fixed.. tak